Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -294,6 +294,24 @@ return
 					},
 					Notes = "Returns whether the specified block type is solid.",
 				},
+				IsSkylightDispersant =
+				{
+					IsStatic = true,
+					Params =
+					{
+						{
+							Name = "BlockType",
+							Type = "number",
+						},
+					},
+					Returns =
+					{
+						{
+							Type = "boolean",
+						},
+					},
+					Notes = "Returns whether skylight may pass through a transparent block unimpeded.",
```

Comment:
"What transparent block"? -> `a block of the specified type` ? Or is it specific for transparent blocks only?